docs/changelog.md

```diff
@@ -22,8 +22,8 @@ use the latest versions of the software.
 | 0.1.z| 1| 1.2.z, 2.0.z, 2.1.z| 0.98.z, 1.0.z, 1.1.z, 1.2.z| 0.9.z, 1.0.0-preZ, 1.0.0| 1.5.z| 5.2.z| 3.2.z| 1.6.z| 2.10.z| 
 | 0.2.z | 1 | 1.2.z, 2.0.z, 2.1.z, 2.2.z, 3.0.z, 3.11.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z, 1.3.z | 0.9.z, 1.0.0-preZ, 1.0.0 | 1.5-1.7.z, 2.3-2.4.z, 5.y, 6.y | 5.2-5.5.z, 6.2-6.6.z, 7.y | 3.2.z | 1.6.z | 2.10.z | 
 | 0.3.z | 2 | 1.2.z, 2.0.z, 2.1.z, 2.2.z, 3.0.z, 3.11.z | 1.0.z, 1.1.z, 1.2.z, 1.3.z, 1.4.z | 1.0.0, 1.1.0, 1.1.2, 1.2.0, 1.3.0, 1.4.0 | 1.5-1.7.z, 2.3-2.4.z, 5.y, 6.y |  5.2-5.5.z, 6.2-6.6.z, 7.y | 3.3.z | 2.2.z | 2.11.z |
-| 0.4.z | 2 | 2.1.z, 2.2.z, 3.0.z, 3.11.z | 1.2.z, 1.3.z, 1.4.z, 2.1.z | 1.3.0, 1.4.0, 1.5.z, 1.6.z, 1.7.z, 1.8.z, 1.9.z, 1.10.z, 1.11.z | 5.y, 6.y | 7.y | 3.4.z | 2.2.z | 2.11.z |
-| 0.5.z | 2 | 2.1.z, 2.2.z, 3.0.z, 3.11.z | 1.2.z, 1.3.z, 1.4.z, 2.1.z | 1.3.0, 1.4.0, 1.5.z, 1.6.z, 1.7.z, 1.8.z, 1.9.z, 1.10.z, 1.11.z | 6.y, 7.y | 7.y | 3.4.z | 2.2.z | 2.11.z | 
+| 0.4.z | 2 | 2.1.z, 2.2.z, 3.0.z, 3.11.z | 1.2.z, 1.3.z, 1.4.z, 2.1.z | N/A | 5.y, 6.y | 7.y | 3.4.z | 2.2.z | 2.11.z |
+| 0.5.z | 2 | 2.1.z, 2.2.z, 3.0.z, 3.11.z | 1.2.z, 1.3.z, 1.4.z, 2.1.z | 1.3.0, 1.4.0, 1.5.z, 1.6.z, 1.7.z, 1.8.z, 1.9.z, 1.10.z, 1.11.z, 1.14.0 | 6.y, 7.y | 7.y | 3.4.z | 2.2.z | 2.11.z | 
```

Comment:
Sorry, I should have clarified that you should add `1.14.z` (wildcard) here, not just 1.14.0 specifically.

Also, adding just 1.14.z here implies that we don't recommend 1.12.z and 1.13.z versions (e.g., they're broken or incompatible with JanusGraph) — if there's nothing wrong with them, shouldn't we add them here as well?

Reply:
gotcha! I only added 1.14.0 as the last version in this repo was for 1.11.0 - do you recommend we test out the versions in between as well? Will update from 1.14.0 to 1.14.z.

Reply:
If you are able to test with 1.12 and 1.13 that would be ideal. If not, we can keep just `1.14.z`.

That said, I don't anticipate any issues with in-between client library versions, as they don't typically change semantics or behavior; it's primarily issues with dependencies or Hadoop profiles are the issues that we may run into.

Reply:
Sounds good. For now, I've updated the `1.14.z` change for the changelog and will test out other versions in parallel to not delay this PR.

Reply:
That's great! We can update the docs separately, and fast-track this PR into the release.

Thanks again for your help!